srv/isc-agent/plugins/tcp/http/models.py

```diff
@@ -0,0 +1,191 @@
+from sqlalchemy import create_engine, MetaData, ForeignKey
+from sqlalchemy import Table, Column, Integer, String, Text, BLOB
+from sqlalchemy.orm import declarative_base, registry
+
+mapper_registry = registry()
+
+Base = mapper_registry.generate_base()
+
+
+# Sigs model
+class Sigs(Base):
+    __tablename__ = 'Sigs'
+
+    id = Column(Integer, primary_key=True)
+    # Should this be Text vs String?
```

Comment:
What's the difference between Text vs String? Is it just the amount of data that it can hold? I would suggest getting with @jullrich to get a sense of what the data will look like and base your decision off of that.